code/game/objects/effects/contraband.dm

```diff
@@ -640,4 +645,33 @@
 	// And yet people still believe...
 	icon_state = "poster37_legit"
 
+//SafetyMoth Original PR at https://github.com/BeeStation/BeeStation-Hornet/pull/1747 (Also pull/1982)
+//SafetyMoth art credit goes to AspEv
+/obj/structure/sign/poster/official/moth_hardhat
+	name = "Safety Moth - Hardhats"
+	desc = "This informational poster uses Safety Moth™ to tell the viewer to wear hardhats in cautious areas. \"It's like a lamp for your head!\""
+	icon_state = "poster38_legit"
+
+/obj/structure/sign/poster/official/moth_piping
+	name = "Safety Moth - Piping"
+	desc = "This informational poster uses Safety Moth™ to tell atmospheric technicians correct types of piping to be used. \"Pipes, not Pumps! Proper pipe placement prevents poor performance!\""
+	icon_state = "poster392_legit"
```

Comment:
"392"?

Honestly, unless it breaks something else, these should just be named `poster_aspev_piping`, etc. This number stuff sucks.

Reply:
The numbers don't do anything because of #24548. I think.

Reply:
oof thats a typo, it used to be a different number in the old PR and i must have missed backspace when i fixed it

Reply:
> 
> 
> The numbers don't do anything because of #24548. I think.

the number stuff is pointless yea, its literally just how the icon states have been named for god knows how long. I'm gonna go Mothblocks' route and just name them poster_aspev_[x]

Reply:
solved, posters now have unique icon states, with non-contraband keeping the _legit statement